packages/paste-design-tokens/tokens/global/text-color.yml

```diff
@@ -93,3 +93,17 @@ props:
   color-text-icon-inverse:
     value: "{!palette-gray-50}"
     comment: Default icon color for inverse backgrounds.
+
+  #availability
+  color-text-available:
+    value: "{!palette-green-60}"
+    comment: Color used to represent an entity or person as "available".
+  color-text-busy:
+    value: "{!palette-orange-60}"
+    comment: Color used to represent an entity or person as "busy".
+  color-text-unavailable:
+    value: "{!palette-red-60}"
+    comment: Color used to represent an entity or person as "unavailable".
+  color-text-offline:
+    value: "{!palette-gray-40}"
+    comment: Color used to represent an entity or person as "offline".
```

Comment:
What background colors is this supposed to be used on? Contrast is only 2.11 on background-body

Reply:
They are likely to be icon only colors, so I guess 3:1 would be beneficial. I don't know where they use them though, I think it must just be the background color to a status dot for them. 

Which if paired with text is technically fine it doesn't have to be 3:1.

Maaaaybe I'll just do backgrounds?

Reply:
I'm going to pull the text colors for as I don't think they are needed and don't even pass icon color contrast ratios